### PR TITLE
[WIP] feat(app): Add LED controller

### PR DESCRIPTION
**Pull request by jhermsmeier**
**ACHTUNG!**

ALLES TURISTEN UND NONTEKNISCHEN LOOKENPEEPERS!
DAS KOMPUTERMASCHINE IST NICHT FÜR DER GEFINGERPOKEN UND MITTENGRABEN!
ODERWISE IST EASY TO SCHNAPPEN DER SPRINGENWERK, BLOWENFUSEN UND POPPENCORKEN MIT SPITZENSPARKEN. IST NICHT FÜR GEWERKEN BEI DUMMKOPFEN. DER RUBBERNECKEN SIGHTSEEREN KEEPEN DAS COTTONPICKEN HÄNDER IN DAS POCKETS MUSS. ZO RELAXEN UND WATSCHEN DER [BLINKENLICHTEN].

[BLINKENLICHTEN]: https://en.wikipedia.org/wiki/Blinkenlights

**Usage:**

To enable, set env `ETCHER_BLINKENLIGHTS` to `1` or `true`

Configuration env vars:

- *Number* `LED_I2C_BUS`
- *Number* `LED_CONTROLLER_ADDRESS`
- *Number* `LED_CONTROLLER_FREQUENCY`
- *Boolean* `LED_CONTROLLER_DEBUG`

For debug output, set env `DEBUG` to `etcher:blinkenlights,pca9685`

### CODE REVIEW

File: lib/gui/app/app.js [hunk 1]
```diff
@@ -410,6 +410,59 @@ app.run(($state) => {
...
+        return
+      }
+
+      // NOTE(jhermsmeier): I have no idea what this might do;
+      // it's straight from the https://github.com/101100/pca9685
+      // example in the README.md
+
+      // Set channel 0 to turn on on step 42 and off on step 255
+      // (with optional callback)
+      pwm.setPulseRange(0, 42, 255, (error) => {
```
Comment (lurch) [inline lib/gui/app/app.js]: Are you missing a `const pwm = lights.driver` or similar?

**Comment by lurch** [comment]
Presumably this'll be rebased on top of #2369 ?

File: lib/gui/app/app.js (hunk 1 above)
Comment (jhermsmeier) [reply lib/gui/app/app.js]: Oh, yes indeed 🤦‍♂️

**Comment by jhermsmeier** [comment]
@lurch yup, this is to get the testing going, and those env vars will evaporate after #2369

**Pull request closed without merging**